packages/core/src/components/onboarding-tip/onboarding-tip.tsx

```diff
@@ -35,7 +35,14 @@ export class OnboardingTip implements ComponentInterface {
     }
 
     if (!this.cleanUp && this.referenceElement && this.tipArrowRef instanceof HTMLElement) {
-      this.cleanUp = OnboardingTip.positionTip(this.referenceElement, this.host, this.tipArrowRef, this.placement);
+      this.cleanUp = positionTooltip({
+        referenceElement: this.referenceElement,
+        tipRef: this.host,
+        tipArrowRef: this.tipArrowRef,
+        placementTip: `${this.placement}-start`,
+        topPositionSmallViewPort: true,
+        halfMainAxisOffset: true,
```

Comment:
@cvanderleij-ilionx Had jij die `forceVisible` (of z'n voorganger) nog niet gepushed?